spec/internal/log/test.log

```diff
@@ -0,0 +1,35 @@
+Parsing SAINT output file...
```

Comment:
this shouldn't be in the repo.

See other plugins using Combustion to have a consistent set of `.gitignore` instructions to prevent this.